even_fib_nums.js

```diff
@@ -10,6 +10,32 @@ function _sumFibs( maxFibValue ) {
 
   // do your work here
 
+  let fibSeq = [0, 1];
+
+  let newArr = [];
```

Comment:
Try doing this without using an array but rather three variables, one to keep track of the current fib number, and two others to keep track of the previous two fib numbers.